docs/ota-client-guide/modules/ROOT/pages/device-monitoring-with-zabbix.adoc

```diff
@@ -0,0 +1,229 @@
+.. Finish the installation.
++
+To sign in to Zabbix, use the following default user name and password:
++
+----
+user: Admin
+pass: zabbix
+----
+
+== Build an OTA image for QEMU
+
+. Clone the manifest file for the quickstart project and download the basic Yocto layers.
++
+For instructions, see https://docs.ota.here.com/getstarted/dev/qemuvirtualbox.html#_create_your_yocto_build_environment[the related section] in the Get Started guide.
+. To set up the Yocto environment, do one of the following:
+** Build a QEMU non-OSTree image.
++
+----
+source meta-updater/scripts/envsetup.sh qemux86-64 <path-to-build-folder> poky
+----
+** Build a QEMU image with a default package manager OSTree.
++
+----
+source meta-updater/scripts/envsetup.sh qemux86-64 <path-to-build-folder> poky-sota-systemd
+----
+. Open your conf/local.conf file and, at the end of the file, add the following lines:
++
+----
+....
+IMAGE_INSTALL_append = " zabbix"
+SOTA_COMM_CONF_ZABBIX_SERVER="10.0.2.2" <1>
+SOTA_COMM_CONF_ZABBIX_SERVERACTIVE="10.0.2.2" <1>
+----
+<1> Substitute the IP address of the system the Zabbix server is running on, as needed. Note that the IP address needs to be hard-coded, so if you are running this on a real device, it is recommended to run the server on something with a static IP.
++
+NOTE: The default host name of the QEMU image is `qemux86-64`.
+
+
+== Run a QEMU image
+
+. Depending on the QEMU image that you built, to make the Zabbix agent visible for the server, do one of the following:
+
+** Run the QEMU non-OSTree image.
++
+----
+../meta-updater/script/run-qemu-ota --uboot-enable=no --host-forward="tcp:0.0.0.0:10555-:10050"
+----
+
+** Run the QEMU image created with the default `OSTree` package manager.
++
+----
+../meta-updater/script/run-qemu-ota --overlay mydevice.cow --host-forward="tcp:0.0.0.0:10555-:10050"
+----
+
+. Install the `zabbix_get` util.
++
+----
+sudo apt-get install zabbix-get
+----
++
+. Check if the Zabbix agent is accessible.
++
+----
+zabbix_get -s localhost -p 10555 -k agent.ping
+----
++
+If `zabbix-agent` is accessible, the command returns `1`
+
+
+== Run a QEMU image in the background mode
+
+Follow this Systemd Unit example:
+----
+  [Unit]
+  Description=QEMU Zabbix Test agent
+  After=network.target networkd.service ntpd.service
+  [Service]
+  Type=simple
+  WorkingDirectory=<absolute path to your qemux86-64 build folder>
+  ExecStart=<absolute path>/meta-updater/scripts/run-qemu-ota --dir <absolute path to your qemux86-64 build folder>/tmp/deploy/images --uboot-enable=no --host-forward="tcp:0.0.0.0:10555-:10050" --no-gui
+  Restart=always
+  RestartSec=5
+  LimitNOFILE=10000
+  [Install]
+  WantedBy=multi-user.target
+----
+
+
+== Add a template and configure a host for Zabbix server
+
+=== Import a Zabbix template
+link:{attachmentsdir}/aktualizr-monitoring-zabbix-template.xml[Download Zabbix template]
```

Comment:
@tkfu could you look to this link? it should download *.xml file, it's unclear for me how to correctly set path to attachment folder. Thanks

Reply:
The way you've done it is correct, for rendering on the docs portal. To get the link to work on github as well, add the following snippet near the top of the file (but below the headline):

```
[NOTE]
====
We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
====
:attachmentsdir: ../assets/attachments
endif::[]
```